### ISSUE TITLE
cannot export the onnx file 

### ISSUE THREAD

**Issue by PredyDaddy** [YunghuiHsu/deepstream-yolo-pose#2]
Hello Yung-Hui,
   I try to use the same code to export the yolov8_pose.onnx file. 
  ```bash
yolo export model=[yolov8s-pose.pt](http://yolov8s-pose.pt/) format=onnx device=0 \
            imgsz=640 \
            half=true \
            dynamic=true \
            simplify=true
```

But I have the error showing below: 
   Ultralytics YOLOv8.0.131 🚀 Python-3.8.12 torch-1.12.0a0+2c916ef CUDA:0 (NVIDIA GeForce RTX 3070, 8191MiB)
YOLOv8s-pose summary (fused): 187 layers, 11615724 parameters, 0 gradients, 30.2 GFLOPs

PyTorch: starting from [yolov8s-pose.pt](http://yolov8s-pose.pt/) with input shape (1, 3, 640, 640) BCHW and output shape(s) (1, 56, 8400) (22.4 MB)

ONNX: starting export with onnx 1.10.1 opset 14...
ONNX: export failure ❌ 0.6s: "slow_conv2d_cpu" not implemented for 'Half'

**Comment by YunghuiHsu** [comment]
Thank you for your help in digging out the problem, I have fixed it in README.md file.

I changed to the following commands to test first to ONNX and then to TensorRT Engine and it's fine.

```
  yolo export model=yolov8s-pose.pt format=onnx device=0 \
              imgsz=640 \
              dynamic=true \
              simplify=true
```

![yolov8s_toonnx](https://github.com/YunghuiHsu/deepstream-yolo-pose/assets/44658525/fa418b87-a55f-4bb8-bece-497ade286fe5)


We can specify FP16 quantization at the "onnx > TensorRT" step.
